docs/README.md

```diff
@@ -8,25 +8,55 @@
 
 # Availability Notify
 
-## Description
-
 `AvailabilityNotfier` is a VTEX Component that shows the availability notify form that is shown when the product isn't available.
```

Comment:
```suggestion
The Availability Notify component is responsible for showing a subscription form when a product SKU isn't available. The form lets customers subscribe to get notified when that item gets restocked. 

The app records the notification request and monitors inventory updates. This way, once the requested SKU gets back in stock, the app will email the shoppers who asked to be notified.
```

Reply:
It would be nice to have an image preview of this component implemented in a store.